devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -171,8 +172,8 @@ private Map getBatteryResult(rawValue) {
 			def pct = batteryMap[volts]
 			result.value = pct
 		} else {
-			def minVolts = 2.4
-			def maxVolts = 2.7
+			def minVolts = (isFirmwareVerLessThan(1, 15, 7)) ? 2.1 : 2.4
```

Comment:
Rather than calling `isFirmwareVerLessThan` multiple times you could call it once and save to a variable.

Reply:
Ha! In my effort to be clean and resourceful, I missed one of the obvious actions! Good catch! Thanks